website/content/partials/chroot-limitations.mdx

```diff
@@ -0,0 +1,3 @@
+!> **Warning:** chroot doesn't copy or recreate directories and files that systemd uses. Specifically on Ubuntu, `/etc/resolv.conf` is a symlink to `/run/systemd/resolve/stub-resolv.conf`, so chroot doesn't copy or link to `resolv.conf`.
```

Comment:
This is technically true but I feel like it's misleading if we refer to "directories and files that systemd uses", because most of the files that it uses _are_ in the default chroot (ex. everything in `/etc` and `/usr`!). What we're not copying -- by default -- are the ephemeral runtime files in `/run`.